pyhf/utils.py

```diff
@@ -157,19 +199,32 @@ def runOnePoint(muTest, data, pdf, init_pars=None, par_bounds=None):
     asimov_mu = 0.0
     asimov_data = generate_asimov_data(asimov_mu, data, pdf, init_pars, par_bounds)
 
-    qmu_v = tensorlib.clip(qmu(muTest, data, pdf, init_pars, par_bounds), 0, max=None)
+    qmu_v = tensorlib.clip(qmu(poi_test, data, pdf, init_pars, par_bounds), 0, max=None)
     sqrtqmu_v = tensorlib.sqrt(qmu_v)
 
     qmuA_v = tensorlib.clip(
-        qmu(muTest, asimov_data, pdf, init_pars, par_bounds), 0, max=None
+        qmu(poi_test, asimov_data, pdf, init_pars, par_bounds), 0, max=None
     )
     sqrtqmuA_v = tensorlib.sqrt(qmuA_v)
 
     CLsb, CLb, CLs = pvals_from_teststat(sqrtqmu_v, sqrtqmuA_v)
 
-    CLs_exp = []
-    for nsigma in [-2, -1, 0, 1, 2]:
-        sqrtqmu_v_sigma = sqrtqmuA_v - nsigma
-        CLs_exp.append(pvals_from_teststat(sqrtqmu_v_sigma, sqrtqmuA_v)[-1])
-    CLs_exp = tensorlib.astensor(CLs_exp)
-    return qmu_v, qmuA_v, CLsb, CLb, CLs, CLs_exp
+    _returns = [CLs]
```

Comment:
Something I'd like you to think about in the review is if the order of returns (below) makes sense/is the most natural.

Reply:
it's not :)

Reply:
Yeah. Do you have thoughts on what to do to improve this? Should we return to the tuple structure of `runOnePoint`'s return: `qmu_v, qmuA_v, CLsb, CLb, CLs, CLs_exp_set`?

Reply:
oh sorry I meant the order I chose did not make sense.. maybe it shoule be  `CLs, CLs_exp, CLsb, CLb, q_mu, qmu_A` .. is this what you have?

Reply:
Ah. Yeah, it is the following:

```python
CLs_obs = pyhf.utils.hypotest(...)
CLs_obs, [CLsb, CLb] = pyhf.utils.hypotest(..., return_p_values=True)
CLs_obs, CLs_exp = pyhf.utils.hypotest(..., return_expected=True)
CLs_obs, [CLs_exp_band] = pyhf.utils.hypotest(..., return_expected_set=True)
CLs_obs, CLs_exp, [CLs_exp_band] = pyhf.utils.hypotest(
    ..., return_expected=True, return_expected_set=True
)
CLs_obs, [CLsb, CLb], CLs_exp, [CLs_exp_band] = pyhf.utils.hypotest(
    ..., return_p_values=True, return_expected=True, return_expected_set=True
)
CLs_obs, [CLsb, CLb], CLs_exp, [CLs_exp_band], [q_mu, q_mu_A] = pyhf.utils.hypotest(
    ...,
    return_p_values=True,
    return_expected=True,
    return_expected_set=True,
    return_test_statistics=True,
)
```

So if there are suggestions on how to make it nicer for a user to keep in their head I'm happy to change it.

Reply:
Whatever we decide, I should probably add all the options to the Hello World notebook as well.

Reply:
@kratsg Suggestions on revisions if you have them is welcomed.

If there are no suggested revisions on the return signature from you two then I will go ahead and cement it with some additional examples to the Hello World notebook (unless people think that is overkill that will clutter up a nice short example).